### ISSUE TITLE
install broken with pip 22 & python 3.7

### ISSUE THREAD

**Issue by oliver-sanders** [pygraphviz/pygraphviz#395]
`pip install pygraphviz` (with no version specified) fails when used with pip version 22 on Python 3.7:

```console
$ pip install pygraphviz
Collecting pygraphviz
  Using cached pygraphviz-1.8.zip (119 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 255
  ╰─> [1 lines of output]
      PyGraphviz requires Python version 3.8 or later (3.7 detected).
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

```

Pip 22 now bails on error (https://github.com/pypa/pip/issues/10655) which means this code:

https://github.com/pygraphviz/pygraphviz/blob/9ec45f388df15252791dc15f6f699bf0b257be7a/setup.py#L6-L12

Will now cause pip to fail whenever `python_version < 3.8` even if a pygraphviz version is not specified.

I think it will be necessary to patch the existing 3.8 package to address this?

**Comment by rossbar** [comment]
This should be fixed by #394 but that requires a new release. @jarrodmillman should we do a patch release?

**Comment by jarrodmillman** [comment]
See 
- https://pypi.org/project/pygraphviz/1.8/
- https://pypi.org/project/pygraphviz/1.9/
